src/main/java/uk/gov/companieshouse/insolvency/delta/consumer/InsolvencyDeltaConsumer.java

```diff
@@ -56,7 +56,12 @@ public void receiveMainMessages(Message<ChsDelta> message,
                                     @Header(KafkaHeaders.OFFSET) String offset) {
         logger.trace(String.format("A new message from %s topic with payload:%s "
                 + "and headers:%s ", topic, message.getPayload(), message.getHeaders()));
-        deltaProcessor.processDelta(message);
+        try {
+            deltaProcessor.processDelta(message);
+        } catch (Exception exception) {
+            logger.error(exception);
```

Comment:
Please log a meaningful message in front of the exception. Something like "Exception while processing the {topic-name} - message in receiveMainMessages method"